pkgs/applications/video/streamlink-twitch-gui/bin.nix

```diff
@@ -39,14 +39,14 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "${basename}-bin";
-  version = "2.1.0";
+  version = "2.2.0";
 
   src = fetchurl {
     url = "https://github.com/streamlink/${basename}/releases/download/v${version}/${basename}-v${version}-${arch}.tar.gz";
     hash =
       if arch == "linux64"
       then
-        "sha256-kfCGhIgKMI0siDqnmIHSMk6RMHFlW6uwVsW48aiRua0="
+        "sha256-XP85M74zPZRnjBfECBmT9Dp3NGcozQ2r6dojRTBGUmw="
       else
         "sha256-+jgTpIYb4BPM7Ixmo+YUeOX5OlQlMaRVEXf3WzS2lAI=";
```

Comment:
Only one checksum got updated

Reply:
Then update the other one your self ^^

Reply:
I don't use Nix, I'm the dev of the application.

Reply:
oh I see, sorry if I was a bit harsh, I thought you were the package maintainer.